pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```
@@ -38,30 +56,20 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
-
-  strictDeps = true;
-
-  buildPhase = ''
-    runHook preBuild
+  ];
 
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
-  '';
+  doCheck = true;
 
-  installPhase = ''
-    runHook preInstall
+  strictDeps = true;
 
+  postInstall = ''
     mkdir -p $out/bin
-    install -Dm444 -t $out/share/emacs/site-lisp/cask *.el *.elc
-    install -Dm555 -t $out/share/emacs/site-lisp/cask/bin bin/cask
-    ln -s $out/share/emacs/site-lisp/cask/bin/cask $out/bin/cask
-
-    runHook postInstall
+    install -Dm755 -t $out/bin $sourceRoot/bin/cask
```

`755` is the default.

```suggestion
    install -D -t $out/bin $sourceRoot/bin/cask
```

---

Maybe in a short future it can be changed to [installBin](https://github.com/NixOS/nixpkgs/pull/332612):
- https://nixpkgs-tracker.ocfox.me/?pr=332612